lib/main.py

```diff
@@ -27,19 +29,27 @@ async def lifespan(_app: FastAPI):
 
 
 APP = FastAPI(lifespan=lifespan)
-TASK_LIST: queue.Queue = queue.Queue(maxsize=100)
 
 
 class BackgroundProcessTask(threading.Thread):
     def run(self, *args, **kwargs):  # pylint: disable=unused-argument
+        nc = NextcloudApp()
+        # Hack to workaround auth problems
+        nc.set_user("admin")
```

Comment:
Imho, we should implement a separate `nc.providers.task_processing.next_task` in the AppAPI that doesn't require a `User` object in the session(without `#[NoAdminRequired]`).
The NC instance may not have an existing user with id = `admin`.